README.md

```diff
@@ -28,11 +30,36 @@ None
 
 ## Example
 
+Basic configuration:
+
+```yaml
+---
+- hosts: all
+  roles:
+    - appsilon.rstudio_connect
+```
+
+Playbook for installing additional Python versions with  [ansible-python-install](https://github.com/Appsilon/ansible-python-install) and configuring RStudio Connect to use them:
+
 ```yaml
 ---
 - hosts: all
+  become: true
+  vars:
+    python_versions:
+      - 3.10.6
+      - 3.7.8
+      - 3.8.6
+    rstudio_connect_enable_python: true
+    rstudio_connect_python_executables:
+      - /opt/python/3.10.6/bin/python3
+      - /opt/python/3.7.8/bin/python3
+      - /opt/python/3.8.6/bin/python3
```

Comment:
Seems ok as it is, but I'm thinking if this could be simplified by removing the `rstudio_connect_python_executables` variable.
From the user point, if I specify that I want those python versions then I shouldn't be copying/pasting the same in another variable. Can't think of a case where the paths will be different than  `/opt/python/***/bin/python3`, especially when we are installing it using our playbook for python. Quick thought: Maybe if the `executables` var is empty then the '/opt/...' path should be used in the for loop in the rstudio_connect.j2. Just for simplicity and to avoid defining too many variables. @koralowiec What do you think?

Reply:
Yeah, I think you are right - it would be better not to repeat. I will try to fix it

Reply:
I did some changes to "simplify" it: [comment](https://github.com/Appsilon/ansible-rstudio-connect/pull/24#pullrequestreview-1098617108)